en/matrix.md

```diff
@@ -0,0 +1,371 @@
+
+    <th class="col-descriptors" style="display: none;" rowspan="2"><a href="https://bitcoinops.org/en/topics/output-script-descriptors/">Descriptors</a></th>
+
+    <th class="col-multipartytxn" style="display: none;color:blue;" colspan="6">Multi-Party Transactions </th>
+
+    <th class="col-pc_silentpayments" style="display: none;color:blue;" colspan="1"><div class="tooltip-container">
+    <a href="https://github.com/bitcoin/bips/blob/master/bip-0047.mediawiki">Payment <br>Codes</a><span class="tooltip">BIP47</span>
+    </div></th>
+
+    <th class="col-pc_silentpayments" style="display: none;color:blue;" colspan="2"><div class="tooltip-container">
+    <a href="https://bitcoinops.org/en/topics/silent-payments/">Silent <br>Payments</a><span class="tooltip">BIP352</span>
+    </div></th>
+
+    <th class="col-lightning" style="display: none;color:blue;" colspan="6"><div class="tooltip-container">Lightning<span class="tooltip">Lightning only sending (Pay Invoice) and receiving (Create Invoice) capabilities are considered.</span>
+    </div></th>
+
+    </tr>
+    <tr>
+
+    <th class="col-platform" style="vertical-align:top;display: none;"><div class="tooltip-container">Device<span class="tooltip">Desktop/Mobile</span>
+    </div></th>
+    <th class="col-platform" style="vertical-align:top;display: none;"><div class="tooltip-container">OS<span class="tooltip">Linux, Windows, OS, Android, iOS</span>
+    </div></th>
+    <th class="col-platform" style="vertical-align:top;display: none;"><div class="tooltip-container">Web /<br> Browser<span class="tooltip">Y/N</span>
+    </div></th>
+
+    <th style="vertical-align:top;"><div class="tooltip-container">
+    <a href="https://bitcoinops.org/en/topics/bech32/">Bech32 <br/><font color="grey">P2WPKH<br/>P2WSH</font></a><span class="tooltip">BIP173</span>
+    </div></th>
+
+    <th style="vertical-align:top;"><div class="tooltip-container">
+    <a href="https://bitcoinops.org/en/topics/bech32/">Bech32m <br/><font color="grey">P2TR</font></a><span class="tooltip">BIP350</span>
+    </div></th>
+
+    <th style="vertical-align:top;">Native <br/>Segwit<br/> Change</th>
+
+    <th class="col-feebumping" style="display: none;"><div class="tooltip-container">
+    <a href="https://bitcoinops.org/en/topics/replace-by-fee/">Full RBF<br><font color="grey">Bump<br>Fee</font></a><span class="tooltip">full-RBF</span>
+    </div></th>
+
+    <th class="col-feebumping" style="display: none;"><div class="tooltip-container">
+    <a href="https://bitcoinops.org/en/topics/replace-by-fee/">Full RBF<br><font color="grey">Cancel<br>TXN</font></a><span class="tooltip">full-RBF</span>
+    </div></th>
+
+    <th class="col-feebumping" style="display: none;"><div class="tooltip-container">
+    <a href="https://bitcoinops.org/en/topics/replace-by-fee/">Full RBF<br><font color="grey">Notification</font></a><span class="tooltip">full-RBF</span>
+    </div></th>
```

Comment:
For these three columns it would also be helpful to learn what they exactly refer to.

I assume it is:

- Bump Fee: "Can use RBF to increase the feerate of a previously submitted transaction"
- Cancel TXN: "Can use RBF to change the outcome of previously submitted transactions, e.g. to cancel a payment"
- Notification: "Tracks when incoming transactions get replaced and notifies the user"

Reply:
Good call.  As you remember, there is a google form which serves as intake for the entries.  On that form, I do have questions similar to what you are proposing.  It totally makes sense to have the same questions (via tooltip) on the resulting matrix.